sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_featureset/feature_schema.py

```diff
@@ -0,0 +1,28 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# pylint: disable=unused-argument,no-self-use
+
+from marshmallow import fields, validate, post_load
+
+from azure.ai.ml._schema.core.schema import PatchedSchemaMeta
+
+
+class FeatureSchema(metaclass=PatchedSchemaMeta):
+    name = fields.Str(
+        required=True,
+        allow_none=False,
+    )
+    type = fields.Str(
+        validate=validate.OneOf(["string", "integer", "long", "float", "double", "binary", "datetime", "boolean"]),
```

Comment:
Is it easier to have a enum type here?

Reply:
in sdk, even though we have enum type, we allow string to be as input